cmd/system-probe/modules/process.go

```diff
@@ -49,11 +51,22 @@ func (t *process) GetStats() map[string]interface{} {
 func (t *process) Register(httpMux *mux.Router) error {
 	var runCounter uint64
 	httpMux.HandleFunc("/proc/stats", func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPost {
+			w.WriteHeader(http.StatusMethodNotAllowed)
+			return
+		}
+
 		start := time.Now()
-		stats, err := t.probe.StatsWithPermByPID()
+		pids, err := getPids(req)
+		if err != nil {
+			log.Errorf("%s", err)
+			w.WriteHeader(http.StatusBadRequest)
+		}
+
+		stats, err := t.probe.StatsWithPermByPID(pids)
 		if err != nil {
 			log.Errorf("unable to retrieve stats using process_tracer: %s", err)
```

Comment:
nit: Can we update this line? The module was originally called `process_tracer` but renamed to just process
```suggestion
			log.Errorf("unable to retrieve stats using process module: %s", err)
```